src/components/table/bl-table.ts

```diff
@@ -0,0 +1,202 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import "element-internals-polyfill";
+import { event, EventDispatcher } from "../../utilities/event";
+import style from "./bl-table.css";
+import BlTableCell from "./table-cell/bl-table-cell";
+import BlTableHeaderCell from "./table-header-cell/bl-table-header-cell";
+import BlTableRow from "./table-row/bl-table-row";
+
+export const blTableTag = "bl-table";
```

Comment:
As per convention, if it's going to be a constant can we rename the variable to `BL_TABLE_TAG` and the rest in this format?

Reply:
all constants are defined as camel case like [here](https://github.com/Trendyol/baklava/blob/next/src/components/checkbox-group/bl-checkbox-group.ts#L9), so we added it in the same way here as well